neurots/extract_input/from_neurom.py

```diff
@@ -14,92 +14,184 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import warnings
```

Comment:
I would say that anything related to the extract_input should be a separate PR (after the initial "trivial" one). Also in my opinion the distributions we extract should be the same independently of the algorithm. So instead of having options here, we should have a default that extracts all info "trunk_simple",  "trunk_with_probability" and what the algorithm will use should depend on the parameter file. This way we will only need to extract the distributions once, and then we can use them for all methods.

Reply:
ok, I can then remove the option to have that, and use it by default. I wanted to hide it in this PR, until it is fully tested for real, etc... and don't loose computational time/disk space to save data we may not need. but as you prefer